pkl/version.go

```
@@ -130,7 +130,10 @@ func (s *semver) isLessThan(other *semver) bool {
 
 func (s *semver) String() string {
 	var builder strings.Builder
-	fmt.Fprintf(&builder, "%d.%d.%d", s.major, s.minor, s.patch)
+	_, err := fmt.Fprintf(&builder, "%d.%d.%d", s.major, s.minor, s.patch)
+	if err != nil {
+		return ""
```

Would it be better to panic here? I can't think of a reason why this would ever fail, but if it did it might be better to do so loudly rather than silently.

---

thanks